### PR TITLE
Add Logger

### PR DESCRIPTION
**Pull request by Kaskat**
Update code style
Add Logback logger

---
This change is [Reviewable](https://reviewable.io/reviews/antheso/bugtracker-backend/2)

### CODE REVIEW

**Review by Antheso — CHANGES_REQUESTED** [review]
> Reviewed 33 of 33 files at r1.
> *[Reviewable](https://reviewable.io/reviews/antheso/bugtracker-backend/2)* status: all files reviewed, 1 unresolved discussion (waiting on @Kaskat, @Kycok-Ivana, and @tanyalev)

___
*[src/main/java/app/Entities/Issue/IssueDao.java, line 54 at r1](https://reviewable.io/reviews/antheso/bugtracker-backend/2#-M1yv5yM3ehA4juSJ8oA:-M1yv5yN2YGE5APn6oOi:b-2fagao) ([raw file](https://github.com/antheso/bugtracker-backend/blob/c9e1ecf38d2f01721efd9a179b30db76127aab5e/src/main/java/app/Entities/Issue/IssueDao.java#L54)):*
> ```
> User assigne
> ```
assignee пишется с двумя е)

File: src/main/java/app/Util/MyLogger.java [hunk 1]
```diff
@@ -0,0 +1,341 @@
+package app.Util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Marker;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class MyLogger implements Logger {
```
Comment (tanyalev) [inline src/main/java/app/Util/MyLogger.java:10]: а почему просто не использовать логгер из log4j?

**Review by tanyalev — APPROVED** [review]

**Comment by Kaskat** [comment]
___
*[src/main/java/app/Entities/Issue/IssueDao.java, line 54 at r1](https://reviewable.io/reviews/antheso/bugtracker-backend/2#-M1yv5yM3ehA4juSJ8oA:-M2-7-QpArZ_J248axwz:b-s9j4n5) ([raw file](https://github.com/antheso/bugtracker-backend/blob/c9e1ecf38d2f01721efd9a179b30db76127aab5e/src/main/java/app/Entities/Issue/IssueDao.java#L54)):*
<details><summary><i>Previously, Antheso (Bohdan) wrote…</i></summary><blockquote>

> ```
> User assigne
> ```
assignee пишется с двумя е)
</blockquote></details>

Поправим

**Comment by Kaskat** [comment]
___
*[src/main/java/app/Util/MyLogger.java, line 10 at r1](https://reviewable.io/reviews/antheso/bugtracker-backend/2#-M1vLCll1dMw8IWXjqRN-r1-10:-M2-7Enj8gc4nK5agZF_:b-7ywxfd) ([raw file](https://github.com/antheso/bugtracker-backend/blob/c9e1ecf38d2f01721efd9a179b30db76127aab5e/src/main/java/app/Util/MyLogger.java#L10)):*
<details><summary><i>Previously, tanyalev wrote…</i></summary><blockquote>

а почему просто не использовать логгер из log4j?
</blockquote></details>

Не работал с такими логерами

Мне они все одинаковые

**Comment by Kaskat** [comment]
___
*[src/main/java/app/Util/MyLogger.java, line 10 at r1](https://reviewable.io/reviews/antheso/bugtracker-backend/2#-M1vLCll1dMw8IWXjqRN-r1-10:-M2-7VBt7DxhIqMuEfdF:b-ljpcqg) ([raw file](https://github.com/antheso/bugtracker-backend/blob/c9e1ecf38d2f01721efd9a179b30db76127aab5e/src/main/java/app/Util/MyLogger.java#L10)):*
<details><summary><i>Previously, Kaskat (Kaskat) wrote…</i></summary><blockquote>

Не работал с такими логерами

Мне они все одинаковые
</blockquote></details>

Тут подключен Logger Logback

**Review by Antheso — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/antheso/bugtracker-backend/2)* status: all files reviewed, 2 unresolved discussions (waiting on @Antheso, @Kycok-Ivana, and @tanyalev)

**Pull request merged**